### PR TITLE
Feature/add scc resource

### PR DESCRIPTION
**Pull request by jamescyeh**

### CODE REVIEW

File: src/main/java/com/bitmovin/api/encoding/captions/ConvertSccCaption.java [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+import com.bitmovin.api.AbstractApiResponse;
+import com.bitmovin.api.encoding.EncodingOutput;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+
+import java.util.Date;
+import java.util.List;
+
+/**
+ * Created by Sandro Linder [sandro.linder@bitmovin.com] on 18/06/2018
```
Comment (gfronza) [inline src/main/java/com/bitmovin/api/encoding/captions/ConvertSccCaption.java]: I would either remove this class doc or change it to your name/email.

File: src/main/java/com/bitmovin/api/encoding/captions/ConvertSccCaptionWebVttSettings.java [hunk 2]
```diff
@@ -0,0 +1,52 @@
+package com.bitmovin.api.encoding.captions;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+
+/**
+ * Created by Sandro Linder [sandro.linder@bitmovin.com] on 19/06/2018
```
Comment (gfronza) [inline src/main/java/com/bitmovin/api/encoding/captions/ConvertSccCaptionWebVttSettings.java]: Same here.

File: src/main/java/com/bitmovin/api/encoding/captions/InputPath.java [hunk 3]
```diff
@@ -0,0 +1,34 @@
+package com.bitmovin.api.encoding.captions;
+
+/**
+ * Created by Sandro Linder [sandro.linder@bitmovin.com] on 18/06/2018
```
Comment (gfronza) [inline src/main/java/com/bitmovin/api/encoding/captions/InputPath.java]: Same here.

File: src/main/java/com/bitmovin/api/encoding/captions/SccCaption.java [hunk 4]
```diff
@@ -0,0 +1,109 @@
+package com.bitmovin.api.encoding.captions;
+
+import com.bitmovin.api.AbstractApiResponse;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+
+import java.util.Date;
+
+/**
+ * Created by Sandro Linder [sandro.linder@bitmovin.com] on 18/06/2018
```
Comment (gfronza) [inline src/main/java/com/bitmovin/api/encoding/captions/SccCaption.java]: Same here.

File: src/main/java/com/bitmovin/api/encoding/captions/StreamCaptionOutputFormat.java [hunk 5]
```diff
@@ -0,0 +1,9 @@
+package com.bitmovin.api.encoding.captions;
+
+/**
+ * Created by Sandro Linder [sandro.linder@bitmovin.com] on 18/06/2018
```
Comment (gfronza) [inline src/main/java/com/bitmovin/api/encoding/captions/StreamCaptionOutputFormat.java]: Same here.

File: src/test/java/com/bitmovin/api/examples/CreateSimpleEncodingWithSCC.java [hunk 6]
```diff
@@ -0,0 +1,414 @@
...
+import org.junit.Test;
+
+import java.io.IOException;
+import java.net.URISyntaxException;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Date;
+
+/**
+ * Created by Roland Kersche on 01.06.17.
```
Comment (gfronza) [inline src/test/java/com/bitmovin/api/examples/CreateSimpleEncodingWithSCC.java]: Same here.

**Review by gfronza — APPROVED** [review]
Looks great. Only a minor detail regarding class documentation. Besides that, maybe add an example for SCC to WebVTT conversion?

**Comment by jamescyeh** [comment]
Will remove credited author (though I did straight up copy them from internal, so I do not feel comfortable attaching my own name to it).  Will add example as well.

**Comment by gfronza** [comment]
LGTM

**Comment by gfronza** [comment]
Still LGTM

**Pull request merged**